manifests/params.pp

```diff
@@ -120,7 +120,7 @@
     $secpcrematchlimit = 1500
     $secpcrematchlimitrecursion = 1500
     $modsec_secruleengine = 'On'
-    if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '7') <= 0 {
+    if $facts['os']['family'] == 'RedHat' and versioncmp($facts['os']['release']['major'], '7') == 0 {
```

Comment:
I'd keep this, since it's no difference at runtime and fewer changes in git. Same below on line 266.